docs/tutorials/mastering.md

```diff
@@ -1,60 +1,626 @@
 # Mastering
 
-In this tutorial, we will deploy and run the SD-Core 5G core network following Control and User Plane Separation (CUPS) principles. The radio and cell phone simulator will also be deployed on an isolated cluster.
-
-```{image} ../images/sdcore_networking.png
-:alt: Network diagram
-:align: center
-```
-
-To complete this tutorial, you will need four machines (virtual or physical), with access to the networks as follows.
-
-| Machine | CPUs | RAM | Disk | Networks |
-| ------- | ---- | --- | ---- | -------- |
-| Control Plane Kubernetes Cluster     | 4 | 8g | 40g | `management` |
-| User Plane Kubernetes Cluster        | 4 | 4g | 20g | `management`, `access`, `core` |
-| Juju Controller + Kubernetes Cluster | 2 | 8g | 40g | `management` |
-| gNB Simulator Kubernetes  Cluster    | 4 | 4g | 20g | `management`, `ran` |
+In this tutorial, we will deploy and run the SD-Core 5G core network in a single server following Control and User Plane Separation (CUPS) principles. The radio and cell phone simulator will also be deployed on an isolated cluster. [Multipass](https://multipass.run/) is used to create separate VMs connected with [LXD](https://ubuntu.com/lxd) networking.
 
+## 1. Prepare Host
 
-## Sample Values
-
-For the purpose of this guide, the following values will be used.
+This tutorial requires:
+- Ubuntu OS is running on host (preferably 22.04 or higher)
+- It has at least one NIC which has internet access
+- The host has free resources: 15 vCPUs, 25 GB RAM and 150 GiB disk 
```

Comment:
```suggestion
- A machine running Ubuntu 22.04 with the following resources:
  - At least one NIC with internet access
  - 15 core
  - 24 GB RAM
  - 150 GiB disk 
```

Reply:
RAM and disk is definitely required, but we could lower the amount of cores to 8, as that will not prevent us from over provisioning the number of vCPUs.

Reply:
I agree @ghislainbourgeois. Although using 25 GB RAM, it's very slow while deploying the charms.

Reply:
CPU requirement is given as 8.

Reply:
Done